### ISSUE TITLE
Cannot edit calendar events when viewing a specific Journal

### ISSUE THREAD

**Issue by dere** [etesync/etesync-web#38]
1. Ensure you have at least one event in at least one journal.
2. Log in to the web app. 
3. The default component shown is an aggregate calendar of all your Calendar-type journals. Click on any event here. It switches to an Event component that displays additional information and a Change History and Edit button.
4. Go Back, then from the hamburger menu select Journals, then any Calendar-type journal with at least one event (such as the journal the event from before is an entry of)
5. The component shown is a calendar of only the specific journal shown. Click on any event here. Notice there is no Change History or Edit button like there was in 3.

**Comment by tasn** [comment]
Yeah, editing (or adding) directly from the journals is currently not supported. It was a combination of lack of time to add it there, and wanting to encourage people to use the main view rather then specific journals view. It's not a bug, but rather a missing feature.
I'm OK with adding it to there as well, but it's not a top priority.
